packages/driver/src/multi-domain/cypress.ts

```diff
@@ -86,6 +90,21 @@ const setup = (cypressConfig: Cypress.Config, env: Cypress.ObjectLike) => {
     specBridgeCommunicator.toPrimary('url:changed', url)
   })
 
+  // outlaw the use of `route` and `server` within the multi-domain context and Cypress.Server.* configurations
```

Comment:
I think these could go in their own file, something like 'unsupported_apis.ts` or something.

Reply:
address in 40574ea3f5